xds/internal/xdsclient/xdsresource/unmarshal_eds.go

```diff
@@ -134,9 +138,13 @@ func parseEDSRespProto(m *v3endpointpb.ClusterLoadAssignment, logger *grpclog.Pr
 			return EndpointsUpdate{}, fmt.Errorf("duplicate locality %s with the same priority %v", lidStr, priority)
 		}
 		localitiesWithPriority[lidStr] = true
+		endpoints, err := parseEndpoints(locality.GetLbEndpoints())
+		if err != nil {
+			return EndpointsUpdate{}, err
```

Comment:
Please wrap this err with more information (see other errors in this function).

Reply:
`parseEndpoints` returns an error which has all the required information.